test/test_document.rb

```diff
@@ -44,6 +44,16 @@ def assert_equal_value(key, one, other)
       assert_equal "foo.bar", @document.data["whatever"]
     end
 
+    should "be able to jsonify its data" do
+      require 'json'
```

Comment:
Sorry for the tone of the bot, @stefanw, in that first comment. I'm sure it didn't mean to tell you off quite so harshly.

You should be able to remove this `require`. From the looks of it, the method above `#to_json` uses `JSON` too, so it should already be required. Were you seeing it break before?